package-lock.json

```diff
@@ -5657,7 +5657,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```

Comment:
These `package-lock` changes aren't required, can we revert them?

Reply:
will do, seemed like this was an automatic change by vs code